### PR TITLE
Changed Tools::displayError to $this->trans

### PR DESCRIPTION
**Pull request by psandmore**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Changed Tools::displayError to $this->trans
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | check code

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
C'mon @psandmore, take a look to the screenshot :/

**Comment by psandmore** [comment]
Mickaël, what screenshot?

**Comment by antoin-m** [comment]
@psandmore I think @mickaelandrieu is talking about @prestonBot 's screenshot

File: controllers/admin/AdminCartsController.php [hunk 1]
```diff
@@ -480,7 +480,7 @@ public function ajaxProcessUpdateQty()
                     $errors[] = $this->trans('You already have the maximum quantity available for this product.', array(), 'Admin.Catalog.Notification');
                 } elseif ($qty_upd < 0) {
                     $minimal_qty = $id_product_attribute ? Attribute::getAttributeMinimalQty((int)$id_product_attribute) : $product->minimal_quantity;
-                    $errors[] = sprintf(Tools::displayError('You must add a minimum quantity of %d', false), $minimal_qty);
+                    $errors[] = sprintf($this->trans('You must add a minimum quantity of %d', array(), 'Admin.Catalog.Notification'), $minimal_qty);
```
Comment (antoin-m) [inline controllers/admin/AdminCartsController.php]: You shouldn't keep a `sprintf` around a `$this->trans`. Please use the second parameter of `$this->trans`

**Review by antoin-m — CHANGES_REQUESTED** [review]

File: controllers/admin/AdminCartsController.php (hunk 1 above)
Comment (psandmore) [inline controllers/admin/AdminCartsController.php]: There is a lot of code that keeps a sprintf around a $this->trans in many places. If it is not allowed, please give me a link to the correct rules. Also PrestaShop team should correct many other places and stop using the code incorrectly. People are looking on PS code.

**Pull request closed without merging**